azuredevops/azuredevops.go

```diff
@@ -77,7 +78,15 @@ func (c *Client) NewRequest(method, URL string, body interface{}) (*http.Request
 // NewBaseRequest does not take into consideration the project
 // and simply uses the base https://%s.visualstudio.com base URL
 func (c *Client) NewBaseRequest(method, URL string, body interface{}) (*http.Request, error) {
-	var buf io.ReadWriter
+	var buf io.Reader
```

Comment:
Could you provide some reasoning behind this please?

Reply:
Thank you for review! 
`NewRequest` ignores `Writer` interface, it takes only `Reader`, so you don't need the `ReadWriter` actually.

https://golang.org/pkg/net/http/#NewRequest